config.py

```diff
@@ -791,11 +791,26 @@
 to **not** specify rejection thresholds for EOG and ECG channels here –
 otherwise, ICA won't be able to "see" these artifacts.
 
+If `None` (default), do not apply automated rejection. If a dictionary,
+manually specify rejection thresholds (see examples).  If `'auto'`, use
```

Comment:
I would call it autoreject_global

more explicit that auto

wdyt?

Reply:
yes